package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "onboard-paulo-sestini",
+  "version": "1.0.0",
+  "description": "A simple server being implemented for learning purpose",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
```

Comment:
Por exemplo:

```json
  "scripts": {
    "start": "node server.js",
    "test": "echo \"Error: no test specified\" && exit 1"
  },
```

O benefício é que pode ser que vc precise mudar esse comando, principalemnte quando vc for inserir o Typescript. Mas mesmo que isso ocorra, sempre seerá `npm start` pra iniciar.